virttest/data_dir.py

```diff
@@ -144,6 +144,26 @@ def get_backend_cfg_path(backend_type, cfg_basename):
     return os.path.join(get_backend_dir(backend_type), 'cfg', cfg_basename)
 
 
+def get_fs_type(abs_path):
```

Comment:
Suggest it being a private function because ideally this should be provided by a utility module (e.g. `filesystem.py`).

Reply:
Is there a way that this is taken a step further and placed in a separate module focused on filesystems (perhaps even an existing one considering the vastness of utilities)?